api/selinuxprofile/v1alpha2/selinuxprofile_types.go

```diff
@@ -14,22 +14,58 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package v1alpha2
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	profilebasev1alpha1 "sigs.k8s.io/security-profiles-operator/api/profilebase/v1alpha1"
 )
 
+const (
+	// AllowSelf describes an "allow" entry meant to give
+	// the same process.
+	AllowSelf = "@self"
+)
+
 // Ensure SelinuxProfile implements the StatusBaseUser interface.
 var _ profilebasev1alpha1.StatusBaseUser = &SelinuxProfile{}
 
+type PolicyRef struct {
+	// The Kind of the policy that this inherits from.
+	// Can be one of the policy objects: SelinuxPolicy or RawSelinuxPolicy.
+	// Or "System" if an already installed policy will be used.
+	// The allowed "System" policies are available in the
+	// SecurityProfilesOpertorDaemon instance.
+	// +kubebuilder:default="System"
```

Comment:
Would it make sense to allow kubebuilder's enum feature to list the allowed kinds?